src/protocol.cpp

```diff
@@ -105,7 +98,6 @@ CMessageHeader::CMessageHeader(const MessageStartChars& pchMessageStartIn, const
     for (; i < COMMAND_SIZE; ++i) pchCommand[i] = 0;
```

Comment:
Can this also be removed?

Reply:
Right, padding is already zero 👍

Reply:
Removed and updated the comment above. Thanks.